src/app/test/api/http/unit/conftest.py

```diff
@@ -44,6 +49,8 @@
         (r"/api/v1/forward/?", ForwardAPI),
         (r"/api/v1/gardens/?", GardenListAPI),
         (r"/api/v1/gardens/(.*)/?", GardenAPI),
+        (r"/api/v1/export/connections/(\w+)/?", GardenConnectionParametersExportAPI),
+        (r"/api/v1/import/connections/(\w+)/?", GardenConnectionParametersImportAPI),
```

Comment:
Picking an appropriate, but still a little random, spot to start the "is this just a PATCH?" conversation I alluded to earlier.

I can see that you somewhat followed the import / export pattern that you previously established with the job import / export.  There's a key distinction here though in that you've made these endpoints operate on a single garden, as where the job endpoints operate on multiple jobs.

By operating on just one garden, it's not really clear to me that there's a meaningful distinction between these new endpoints and the existing ones.  That is, `/api/v1/export/connections/somegarden` is really just returning a subset of the same data you could already get via `/api/v1/gardens/somegarden`.  And `/api/v1/import/connections/somegarden` is just a more limited version of the existing `PATCH` you could already do.

A couple of questions that I think we need to answer:
* Is there functionality / value added by the single garden version of these import / export endpoints that I'm not seeing?
* Whether there is or not, does it make more sense have these operate on multiple gardens similar to how the job import / export works?  I'm inclined to say yes, but if you went down this current path for a specific reason then we certainly can discuss it.

Reply:
Point taken and I noticed that the new endpoints weren't adding very much to the mix. But:
* The reason a new endpoint was added -- vice reusing the patch code -- was because a colleague who has since moved on decided that `.../api/v1/export/...` and `.../api/v1/import/...` constituted good API design and that import/export command should live there. I followed suit both originally and in what I expect is the first in a series of import/export endpoints. Regardless, having all export functions under an `.../export/...` path *feels* like a better design than having some export functions in one place, others in another. Consistency etc.
* As far as the UI goes, it needs a mechanism to export and import only a single garden. It was agreed that the ultimate buttons that will make it into the UI will be on the specific garden's config page.
* Alternately (or additionally), there *could* be an "export all garden connection configs" button, but I recall that you thought at that time of the discussion that the re-import of a list of configs would be problematic. I think the argument was that the name of a garden is the only discriminator. If the UI always gets back a list of garden configs when exporting or updating, that's the same problem as far as I can tell.